internal/servers/testing.go

```diff
@@ -0,0 +1,38 @@
+package servers
+
+import (
+	"context"
+	"crypto/rand"
+	"testing"
+
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/kms"
+	"github.com/hashicorp/boundary/internal/types/scope"
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/require"
+)
+
+// TestRepo creates a repo that can be used for various purposes.
+// Crucially, it ensures that the global scope contains a valid root
+// key.
+func TestRepo(t *testing.T, conn *gorm.DB, rootWrapper wrapping.Wrapper) *Repository {
```

Comment:
Hum... I'm not sure we continued using this pattern after iam.TestRepo(...)  I think all the other repos are just created by following a pattern of getting a conn, wrapper, kmsCache and then creating the repo.  In this case, I think there's only one current reference...  perhaps we don't need this?